view/checkin/checkin/search.phtml

```diff
@@ -1,3 +1,20 @@
+<?php
+$this->inlineScript()
+    ->appendFile('/js/instascan.min.js')
+    ->appendFile('/js/readqr.js');
+?>
+<div id="qr" style="display:none" class="panel panel-default">
+    <div class="panel-heading display-table credit-card-box" >
```

Comment:
Why is there a white space before the tag closing character? There's none above, but one in many below, what's your code style on that issue?

Reply:
There isn't really a code style for HTML, they've probably come about during code editing